README.md

```diff
@@ -1,2 +1,14 @@
 # mantle-shipengine
 ShipEngine
+Integration with ShipEngine for shipping rates, create labels, and void label. In this Integration we worked on staging environment. So, the rates may vary from production evironment.
```

Comment:
Readme file is still not updated as per suggestions.

Reply:
Issue resolved successfully.